PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java

```diff
@@ -22,6 +24,8 @@
     private Button startButton;
     private Button newUserButton;
     private Button aboutButton;
+    private int alert_icon = R.drawable.ic_launcher;
```

Comment:
@top-gun007 Could you please elaborate on this and let us know above which version is it deprecated?